packages/aws-cdk-lib/core/lib/cfn-resource.ts

```diff
@@ -640,6 +640,16 @@ const MERGE_EXCLUDE_KEYS: string[] = [
   'Fn::Or',
 ];
 
+// Availabe wafv2 empty objects
```

Comment:
This description is vague and doesn't explain what these are used for, nor how "empty objects" relates to `DELETE_MERGE_EXCLUDE_KEYS`. Maybe provide a short snippet of context, you can reference the description for `MERGE_EXCLUDE_KEYS` on line 617